pkg/wharfapi/token.go

```diff
@@ -11,7 +11,6 @@ type Token struct {
 	TokenID    uint   `json:"tokenId"`
 	Token      string `json:"token"`
 	UserName   string `json:"userName"`
-	ProviderID uint   `json:"providerId"`
```

Comment:
Oh yea this was removed from the database a while ago as it created a circular dependency between the token and the provider.

Do you mind also updating the CHANGELOG.md?

Reply:
Nope. You probably forgot to push

Reply:
Trial 2 on this up now. I verified i can see the Changelog changes on this PR.